addons/common/functions/fnc_getFewestGroupsHC.sqf

```diff
@@ -0,0 +1,51 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Returns HC with fewest groups, if available.
```

Comment:
"Returns the headless client with the fewest local groups. objNull if none are available."

Reply:
dd03544